2817-starter-quick-start (#2940)
pwa-builder/PWABuilder: docs/starter/quick-start.md

# PWA Starter

The pwa-starter is a starter codebase, just like create-react-app or the Angular CLI can generate, that 
- Uses the PWABuilder team's preferred front-end tech stack
- Is specifically for building cross-platform Progressive youb Apps (PWAs)
- And enforces best practices from the start

If you're new to developing PWAs and aren't sure where to start, check out this [blog series.]()

## Prerequisites

There are just a few things you will need before getting started:

- A Github Account, which you can create at [GitHub](https://github.com/).
- A code editor, such as [Visual Studio Code](https://code.visualstudio.com/).
- A web browser, such as [Microsoft Edge](https://www.microsoft.com/en-us/edge).
- [Install Node.js and Git](https://docs.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-windows)

## Getting the Starter

> **Note**: You can also use our VSCode Extension, PWA Studio, to generate a starter project. Go [here]() for more info.

Reviewer: ```suggestion
?> **Note** You can also use our VSCode Extension, PWA Studio, to generate a starter project. Go [here](#/studio/quick-start) for more info.
```

> First, visit [the PWA Starter on Github](https://aka.ms/pwa-starter). Now, click the `Use this template` button.
> Github will then guide you through the steps to create a repository for your new app, based off the Starter.
> 
> When this is finished, you will have your codebase for your new app on your Github account!

Reviewer: ```suggestion
To use the starter:

1. Visit the [PWA Starter repository.](https://github.com/pwa-builder/pwa-starter)

2. Click the green `Use this template` button.

3.  Provide a repository name and other options you wish to include.

4. Click `Create repository from template`.

5. You will be redirected to your new repository!

Next, you just need to clone the repository to start developing. If you need help, check out these [docs.]()
```

> ## Starting Development
> 
> Now, using the code repository we created above, you can start developing your app.
> 
> First, choose the local option:
> 
> <div class="docs-image">
>   <img src="/assets/starter/quick-start/local-button.png" alt="Image of PWA Studio commands being filtered in VS Code" width=600/>
> </div>
> 
> Next, click the copy button:
> 
> <div class="docs-image">
>   <img src="assets/starter/quick-start/copy-button.png" alt="A screenshot that shows the copy button on the pwa-starter Github repo" width=600 />
> </div>
> 
> After you have our URL copied, open a terminal (Windows Terminal comes by default in Windows 11, but Powershell or any other terminal on your operating system of choice will work) and type `git clone` followed by the URL you just copied. ``
> 
> This will clone a copy of the pwa-starter to our device!
> 
> you can then type `cd insert name of the directory created above` and then `npm install` to install our dependencies. you are now ready to start building!
> 
> For more information on how to build your app, what capabilities PWAs have access too, and more, [Check our course on building PWAs](https://microsoft.github.io/win-student-devs/#/30DaysOfPWA/core-concepts/)

Reviewer: ```suggestion
```
